src/main/java/bio/terra/cli/command/config/set/ResourceLimit.java

```diff
@@ -19,12 +19,13 @@
   static class ResourceLimitArgGroup {
     @CommandLine.Option(
         names = "--max",
-        description = "maximum number to allow before throwing an error")
+        description = "Maximum number to allow before throwing an error.")
     private int max;
 
     @CommandLine.Option(
         names = "--default",
-        description = "use the default number of resources: " + Config.DEFAULT_RESOURCES_CACHE_SIZE)
+        description =
```

Comment:
indicate how this works with `--max`

Reply:
Similar to above, you can only specify one at a time and picocli indicates this relationship for us in the auto-generated usage help:
```
> terra config set resource-limit
Error: Missing required argument (specify one of these): (--max=<max> | --default)
Usage: terra config set resource-limit (--max=<max> | --default)
```